_posts/2015-06-23-hardly-anyone-knows-cd.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Hardly Anyone Knows Continuous Delivery
```

Comment:
I think your version implies that most people do, which is not true. Honestly, it's meant to be a bit abrupt.